test/helpers/name.go

```diff
@@ -34,3 +44,46 @@ func ObjectPrefixForTest(t *testing.T) string {
 func ObjectNameForTest(t *testing.T) string {
 	return AppendRandomString(ObjectPrefixForTest(t))
 }
+
+// AppendRandomString will generate a random string that begins with prefix.
+// This is useful if you want to make sure that your tests can run at the same
+// time against the same environment without conflicting.
+// This method will use "-" as the separator between the prefix and
+// the random suffix.
+// This method will seed rand with the current time when the package is initialized.
```

Comment:
This is not true.

Reply:
I moved this function from the file I deleted, and did not make any change.
If what you meant was seed rand is NOT done by this method, I've removed this line, and added the comment as per your last comment.